src/components-examples/cdk/drag-drop/cdk-drag-drop-custom-clone-target/cdk-drag-drop-custom-clone-target-example.html

```diff
@@ -0,0 +1,7 @@
+<div cdkDropList class="example-list" (cdkDropListDropped)="drop($event)">
+  <div class="example-box" *ngFor="let movie of movies" cdkDrag [cdkDragCloneTarget]="cloneTarget">
+    {{movie.title}}
+  </div>
+</div>
+
+<div #cloneTarget></div>
```

Comment:
I'm not sure whether this example would illustrate the difference very well. Maybe there should be some inherited styles so it's more visible?

Reply:
I actually thought of doing that then thought maybe it was overkill. Sure thing!

Reply:
this is done now :-)